### PR TITLE
Get available currencies + formatting info from server; format currency strings in `balances`

### PR DESCRIPTION
**Pull request by zapu**
We are now able to do:
```
Balances for account GB6BRS2WFYPPI4PQNYAWYZ3CRNJHIHWNZ67T5GTSZXQYV2DC3BGGJD2G (Primary):
XLM	3.0372700 (PLN ~4.6471909 zł)

Balances for account GD2BY6UI4JK5STWDC2B7AAQ6GI73LPJI4XSHGIX2ZT44E5FBPWGSWFGZ (Primary):
XLM	1.2827000 (EUR ~€0.4605038)
```

### CODE REVIEW

**Comment by zapu** [comment]
@mlsteele this is the currency formatting thing I've been working on last week. Should we try to land any of this or wait till frontend gets to it?

File: go/client/cmd_wallet_balances.go [hunk 1]
```diff
@@ -98,7 +98,14 @@ func (c *cmdWalletBalances) runForUser(cli stellar1.LocalClient) error {
...
 					if err == nil {
-						localAmountStr = fmt.Sprintf(" (~%s %s)", localAmount, string(acc.ExchangeRate.Currency))
+						ret, err := cli.FormatCurrencyString(context.Background(), stellar1.FormatCurrencyStringArg{
+							Money: localAmount,
+							Code:  acc.ExchangeRate.Currency,
+						})
+						if err != nil {
+							c.G().Log.Warning("Unable to format amount in: %s", err)
+						}
+						localAmountStr = fmt.Sprintf(" (%s ~%s)", string(acc.ExchangeRate.Currency), ret)
```
Comment (mlsteele) [inline go/client/cmd_wallet_balances.go]: This seems like a great way to make sure the system is working. But I don't think it looks better with both the code and the symbol. Just the code before seemed fine, especially given there's a squiggle. This version uses `ret` even if it errored out.

File: go/libkb/interfaces.go [hunk 2]
```diff
@@ -620,6 +621,9 @@ type Stellar interface {
 	CreateWalletSoft(context.Context)
 	Upkeep(context.Context) error
 	OnLogout()
+
+	SetServerDefinitions(context.Context, stellar1.StellarServerDefinitions) error
+	GetServerDefinitions(context.Context) (stellar1.StellarServerDefinitions, error)
```
Comment (mlsteele) [inline go/libkb/interfaces.go]: SetServerDefinitions shouldn't need to be exported. If there was a checked type cast to `*Stellar` in `RefreshServerConfig` that would be fine.

Comment (zapu) [reply go/libkb/interfaces.go]: makes sense, thanks!

**Pull request merged**